lib/std/special/build_runner.zig

```diff
@@ -189,6 +207,9 @@ fn usage(builder: *Builder, already_ran_build: bool, out_stream: anytype) !void
         \\  -h, --help                  Print this help and exit
         \\  --verbose                   Print commands before executing them
         \\  -p, --prefix [path]         Override default install prefix
+        \\  --output-lib-dir [path]     Override default library directory path
+        \\  --output-exe-dir [path]     Override default executable directory path
+        \\  --output-include-dir [path] Override default include directory path
```

Comment:
Maybe we should mention that these are relative to the install prefix

Reply:
https://github.com/ziglang/zig/blob/b8588f7ab4458c21d105115c2fc8a2acbb456c9a/lib/std/build.zig#L202-L204
the paths are not relative to the install prefix

the default include and exe dir values follow the FHS so I don't think changing the directory relative to the prefix is worthwhile

though the lib dir is rather ambigious because of the optional quad and that maybe should have its own flag

Reply:
fair point

I've pushed code that takes dest_dir into consideration when specifying the directories though I am sure the code could be improved since I am not well versed in Zig

Reply:
How about we support both absolute and relative paths. If a relative path is passed it is considered relative to the install prefix.

Reply:
That is a fair expectation.
Added it to the code